Source/Core/DolphinWX/FrameTools.cpp

```diff
@@ -292,6 +292,70 @@ void CFrame::OpenGeneralConfiguration(wxWindowID tab_id)
   m_main_config_dialog->SetFocus();
 }
 
+void CFrame::CycleProfile(bool forward) {
```

Comment:
Brace goes on the next line in our coding style

Reply:
Serves me right for trying to lint by eye. Fixed.